assets/app/features/upgrade/upgrade.constants.js

```diff
@@ -12,5 +12,26 @@
             'ha': ['clusters_healthy'],
             'ceph': ['ceph_healthy']
         })
-        .constant('ADMIN_UPGRADE_TIMEOUT_INTERVAL', 1000);
+        .constant('PREPARE_TIMEOUT_INTERVAL', 1000)
+        .constant('ADMIN_UPGRADE_ALLOWED_DOWNTIME', 5 * 60 * 1000)
```

Comment:
@vuntz suggested to change these to poll every 5-10sec and tolerate 20-30min downtime. PR updated.